Muart-C/politico
# #164059336 validate empty name

**What does this PR do?**
- Add a validation function for a name provided for patching

**How should the app be tested?**
-  `git clone https://github.com/Muart-C/politico.git`

- `cd politico`

- `git checkout ch-validate-empty-name-164059336`

-  `$ export FLASK_APP="run.py"`
   `$ export FLASK_ENV="development"`
   `$ export FLASK_DEBUG=1`
   `$ export DATABASE_POLITICO="politico"`
   `$ export DATABASE_USERNAME="yourpostgresusername"`
   `$ export DATABASE_PASSWORD="yourpostgrespassword"`
   `$ export DATABASE_HOST="localhost"`
   `$ export DATABASE_PORT=5432`
 - Or alternatively you can create a .env file and add the above configurations on your local machine.

-  Then start your app via the command `flask run` 

- First of all, ensure you install Postman - Test client to test the API endpoint.

[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/4e6c88cc481fc0ceefd0) via your web browser.

**Steps to follow on postman**

- Send a POST request to create a party.
- Then send a POST request with a name attribute that is blank or number.
- The validation function notes that the name is invalid and adds returns the necessary error code and message. 

**What are the relevant pivotal tracker stories**
https://www.pivotaltracker.com/story/show/164059336
